syncstorage-mysql/src/diesel_ext.rs

```diff
@@ -3,11 +3,43 @@ use std::{fmt::Debug, marker::PhantomData};
 use diesel::{
     backend::Backend,
     insertable::CanInsertInSingleQuery,
+    mysql::Mysql,
     query_builder::{AstPass, InsertStatement, QueryFragment, QueryId},
+    query_dsl::methods::LockingDsl,
     result::QueryResult,
     Expression, QuerySource, RunQueryDsl,
 };
 
+/// Emit MySQL <= 5.7's `LOCK IN SHARE MODE`
+///
+/// MySQL 8 supports `FOR SHARE` as an alias (which diesel natively supports)
```

Comment:
Could you add an additional comment here that this is needed for MariaDB compat?

Reply:
done